### ISSUE TITLE
Low sensitivity of explicit-types

### ISSUE THREAD

**Issue by vladyan18** [protofire/solhint#492]
The explicit-type rule is not checked when casting a variable.

## Example

Option: `explicit`

```solidity
function test() external {
  uint256 a = uint(10);
}
```

Expected: error because of `uint(10)`

Observed: no errors

**Comment by dbale-altoros** [comment]
@vladyan18 that's a good catch
I'll try to fix it in next release

Feel free to submit a PR to correct this
Much appreciated

Thanks for posting!

**Comment by vladyan18** [comment]
> Feel free to submit a PR to correct this Much appreciated

Already done :)

https://github.com/protofire/solhint/pull/493
